mlx/utils.cpp

```diff
@@ -1,6 +1,8 @@
 // Copyright © 2023 Apple Inc.
 
+#include <limits>
 #include <sstream>
+#include <type_traits>
```

Comment:
Do we need these added includes here?

Reply:
You're correct, these are unnecessary and have been removed.